doc/building.rst

```diff
@@ -152,62 +152,81 @@ Build Type and Logging
      - 200: Debug
      - 100: Trace
 
-UA_BUILD_* group
-^^^^^^^^^^^^^^^^
+Select build artefacts
+^^^^^^^^^^^^^^^^^^^^^^
 
-By default only the shared object libopen62541.so or the library open62541.dll
-and open62541.dll.a resp. open62541.lib are build. Additional artifacts can be
-specified by the following options:
+By default only the main library shared object libopen62541.so (open62541.dll)
+or static linking archive open62541.a (open62541.lib) is built. Additional
+artifacts can be specified by the following options:
 
 **UA_BUILD_EXAMPLES**
-   Compile example servers and clients from :file:`examples/{xyz}.c`. A static and a dynamic binary is linked, respectively.
+   Compile example servers and clients from :file:`examples/*.c`.
 
 **UA_BUILD_UNIT_TESTS**
-   Compile unit tests with Check framework. The tests can be executed with ``make test``
-
-**UA_BUILD_EXAMPLES_NODESET_COMPILER**
-   Generate an OPC UA information model from a nodeset XML (experimental)
+   Compile unit tests. The tests can be executed with ``make test``
 
 **UA_BUILD_SELFSIGNED_CERTIFICATE**
    Generate a self-signed certificate for the server (openSSL required)
 
-UA_ENABLE_* group
-^^^^^^^^^^^^^^^^^
-
-This group contains build options related to the supported OPC UA features.
+Detailed SDK Features
+^^^^^^^^^^^^^^^^^^^^^
 
 **UA_ENABLE_SUBSCRIPTIONS**
    Enable subscriptions
+
 **UA_ENABLE_SUBSCRIPTIONS_EVENTS**
     Enable the use of events for subscriptions
+
 **UA_ENABLE_METHODCALLS**
    Enable the Method service set
+
 **UA_ENABLE_NODEMANAGEMENT**
    Enable dynamic addition and removal of nodes at runtime
+
 **UA_ENABLE_AMALGAMATION**
    Compile a single-file release into the files :file:`open62541.c` and :file:`open62541.h`
+
 **UA_ENABLE_MULTITHREADING**
    Enable multi-threading support
+
 **UA_ENABLE_IMMUTABLE_NODES**
-   Nodes in the information model are not edited but copied and replaced. The replacement is done with atomic operations so that the information model is always consistent and can be accessed from an interrupt or parallel thread (depends on the node storage plugin implementation). This feature is a prerequisite for ``UA_ENABLE_MULTITHREADING``.
+   Nodes in the information model are not edited but copied and replaced. The
+   replacement is done with atomic operations so that the information model is
+   always consistent and can be accessed from an interrupt or parallel thread
+   (depends on the node storage plugin implementation). This feature is a
+   prerequisite for ``UA_ENABLE_MULTITHREADING``.
+
 **UA_ENABLE_COVERAGE**
    Measure the coverage of unit tests
 
+**UA_NAMESPACE_ZERO**
+
+   Namespace zero contains the standard-defined nodes. The full namespace zero
+   may not be required for all applications. The selectable options are as follows:
```

Comment:
Shouldn't here be minimal, reduced, full? (At least it's like this in the CMake)